services/validationService.js

```diff
@@ -177,7 +181,7 @@ export default async function validate(context, req) {
                 status: 500,
                 headers: { 'Content-Type': 'application/json' },
                 body: {
-                    feedback: 'unhandled server error please contact the iati technical team',
+                    feedback: 'An unexpected server error occurred. Please contact IATI Secretariat Support with code 01.',
```

Comment:
Is this code documented or listed anywhere? If we got a message with a code I'd want a place to look up what it meant. Maybe just miss the code? The rest of the message rewrite is good tho.

Reply:
Okay, will do. I added the code because there are multiple places where this message is returned in response to an exception, and so without this it makes debugging difficult (it attempts to include the received error message, but not infrequently this is undefined or empty).

Reply:
Hmmmmmm ....... can there be a text message describing the place the error came from in a few technical words? Then if a dev gets it they can copy & search the code base for it and if a human reports a slightly garbled version we can hopefully still work it out.

Reply:
Yeah I suppose, I'll have a look.